test/sharness/t0053-dag.sh

```diff
@@ -268,6 +268,41 @@ test_dag_cmd() {
     test_cmp resolve_obj_exp resolve_obj &&
     test_cmp resolve_data_exp resolve_data
   '
+
+  test_expect_success "dag stat of simple IPLD object" '
```

Comment:
Added a few tests here, do we need more before shipping this?

Reply:
This seems reasonable.